.travis.yml

```diff
@@ -2,7 +2,7 @@ language: node_js
 before_install:
   - npm install -g npm@latest
 install:
-  - npm install --global electron
+  - npm install --global electron@1.4.4
```

Comment:
@jviotti @lurch To be honest I'd rather have Electron as an "external" dependency and mention it in the README, as having several repositories with Electron as a dev dependency tends to needlessly fill up disk space with the 140MB+ Electron comes with, and in most cases `npm run test-node` locally should cut it